js/provider/taihe.js

```diff
@@ -0,0 +1,313 @@
+/* eslint-disable no-unused-vars */
+/* global async getParameterByName forge */
+function build_taihe() {
+  const axiosTH = axios.create({
+    baseURL: 'https://music.taihe.com/v1',
+  });
+  axiosTH.interceptors.request.use((config) => {
+    const params = {...config.params};
+    params.timestamp = 1614872801;// Math.round(Date.now() / 1000);
```

Comment:
fixed timestamp?

Reply:
This is for debugging. It should not be a hardcoded timestamp though the server doesn't verify it now.